docs/build/index.md

```diff
@@ -0,0 +1,8 @@
+<a id='Main.Lapidary-1'></a>
+
+<a href='#Main.Lapidary-1'> # </a>**Constant**
```

Comment:
This should be `**Module**`, needs an additional def in https://github.com/MichaelHatherly/Lapidary.jl/blob/7cb5b2ee8b3fd23b91c47b2acbc7dee471355e61/src/utilities.jl#L110-L114. (You can add it if you want, or just leave it for another commit later.)